docs/oauth2-examples-multiresource.md

```diff
@@ -0,0 +1,174 @@
+
+## AMQP clients and management users registered in same OAuth 2.0 server but with different audience
+
+RabbitMQ is configured with two OAuth2 resources one called `rabbit_prod` and another `rabbit_dev`. Think that the
+production team refers to RabbitMQ with the `rabbit_prod` audience. And the development team with the `rabbit_dev` audience.
+Because both teams are registered in the same OAuth2 server you are going to configure its settings such as `jwks_url` at the
+root level so that both resources share the same configuration.
+
+In the past, RabbitMQ imposed a restriction where the scopes had to be prefixed with the name of the resource/audience. For instance, if `resource_server_id` was `rabbitmq1`, all scopes had to be prefixed with the value `rabbitmq1` e.g. `rabbitmq1.tag:administrator`.
+
+Since RabbitMq 3.11, you can configure the scope's prefix independent from the resource_id/audience. This is exactly what this scenario uses which configures the scope prefix with the value `rabbitmq.` so that all scopes, regardless of the resource, have the same prefix.
+
+
+### Test applications accessing AMQP protocol with their own audience
+
+This is a summary of the configuration, found in [rabbitmq.conf](https://github.com/rabbitmq/rabbitmq-oauth2-tutorial/tree/support-multiple-resource-server-ids/conf/multi-keycloak/rabbitmq.conf):
+
+- There are two OAuth2 clients (`prod_producer` and `dev_producer`) declared in keycloak and configured to access their respective audience: `rabbit_prod` and `rabbit_dev`
+- RabbitMQ OAuth2 plugin has been configured with 2 resources: `rabbit_prod` and `rabbit_dev`
+<pre class="lang-ini">
+auth_oauth2.resource_servers.1.id = rabbit_prod
+auth_oauth2.resource_servers.2.id = rabbit_dev
+</pre>
+- Also RabbitMQ OAuth2 plugin has been configured with common settings for the two resources declared above
+<pre class="lang-ini">
+auth_oauth2.preferred_username_claims.1 = preferred_username
+auth_oauth2.preferred_username_claims.2 = user_name
+auth_oauth2.preferred_username_claims.3 = email
+auth_oauth2.jwks_url = https://keycloak:8443/realms/test/protocol/openid-connect/certs
+auth_oauth2.scope_prefix = rabbitmq.
+auth_oauth2.https.peer_verification = verify_peer
+auth_oauth2.https.cacertfile = /etc/rabbitmq/keycloak-cacert.pem
+</pre>
+
+Follow these steps to deploy Keycloak and RabbitMQ:
+
+
+- Launch Keycloak. Check out [Admin page](http://localhost:8081/admin/master/console/#/test) with the credentials `admin:admin`
+
+    ```
+    make start-keycloak
+    ```
+
+- Launch RabbitMQ
+
+    ```
+    export MODE="multi-keycloak"
+    make start-rabbitmq
+    ```
+
+- Launch AMQP producer registered in Keycloak with the <b>client_id</b> <code>prod_producer</code> and with the permission to access `rabbit_prod` resource and with the scopes `rabbitmq.read:*/* rabbitmq.write:*/* rabbitmq.configure:*/*`
```

Comment:
Add "the" after "Launch"
Add a comma after prod_producer
Add "the" after "access"
add a comma after the word "resource" 

So this is the rewrite:

 Launch the AMQP producer registered in Keycloak with the <b>client_id</b> <code>prod_producer</code>, and with the permission to access the `rabbit_prod` resource, and with the  scopes `rabbitmq.read:*/* rabbitmq.write:*/* rabbitmq.configure:*/*`.